test/parallel/test-child-process-validate-stdio.js

```diff
@@ -30,6 +30,19 @@ common.expectsError(() => _validateStdio(stdio2, true),
                     { code: 'ERR_IPC_SYNC_FORK', type: Error }
 );
 
+// should throw if stdio is not a valid input
+{
+  const stdio3 = ['foo'];
```

Comment:
Just name these variables `stdio`. One of the reasons for block scoping each test is to avoid numbering our variables.